### PR TITLE
Cron jobs documentation

### PR DESCRIPTION
**Pull request by gimenete**
I haven't included any screenshot yet.

### CODE REVIEW

File: src/docs/reference/cron-jobs.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  `0 */2 * * *`
+
+- Run a command every 2nd day of the month at 6:00 AM:
+
+  `0 6 2 * *`
+
+## Service execution
+
+Scheduled services should exit as soon as they are done with the task they are responsible to perform. Thus, the process should close any connections, such as database connections, to exist properly.
```
Comment (FarazPatankar) [inline src/docs/reference/cron-jobs.md:85]: `exit properly`?

Comment (gimenete) [reply src/docs/reference/cron-jobs.md:85]: Good catch :D

**Review by FarazPatankar — COMMENTED** [review]
We should also add the banner for priority boarding on the docs page as this is in beta. Example -> https://github.com/railwayapp/docs/blob/main/src/docs/reference/public-api.md?plain=1#L5

**Comment by gimenete** [comment]
@FarazPatankar done!

**Pull request merged**